### ISSUE TITLE
Don't hardcode max number of team pages

### ISSUE THREAD

**Issue by nwalters512** [the-blue-alliance/the-blue-alliance-android#490]
https://github.com/the-blue-alliance/the-blue-alliance-android/blob/master/android/src/main/java/com/thebluealliance/androidclient/adapters/TeamListFragmentPagerAdapter.java#L14

This is actually relevant now that we've run over the current max team number of 5999. Handling this properly will be interesting, because the number of teams we have in the database could change if we load in additional teams while the team lists are showing. Observable magic to the rescue? Or fire something on the EventBus when new teams are loaded and then check if we have to recompute the tabs? Open to ideas on the best way to handle this.

**Comment by phil-lopreiato** [comment]
I don't think we really need to live-update the tabs because it'll be tricky, like you said. As long as the data gets downloaded (which I think it does currently), we can just show however many tabs we need. We'll probably have to pass a total team count via the adapter constructor.
